.vscode/launch.json

```diff
@@ -0,0 +1,34 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+      {
+          "type": "pwa-chrome",
+          "request": "launch",
+          "name": "Launch Chrome against localhost",
+          "url": "http://localhost:4200",
+          "webRoot": "${workspaceFolder}"
+      },
+      {
+        "type": "chrome",
+        "request": "launch",
+        "name": "Debug karma tests",
+        "url": "http://localhost:9876/debug.html",
+        "webRoot": "${workspaceFolder}",
+        "runtimeArgs": [
+          "--headless"
+        ],
+        "sourceMaps": true,
+        "sourceMapPathOverrides": {
+          "webpack:/*": "${webRoot}/*",
+          "/./*": "${webRoot}/*",
+          "/src/*": "${webRoot}/*",
+          "/*": "*",
+          "/./~/*": "${webRoot}/node_modules/*"
+        },
+        "port": 9223,
+      }
+  ]
+}
```

Comment:
I don't usually use vs-code's inbuilt debugger for frontend-projects; Cannot vouch for that. But looks good to me!

Reply:
I mostly use this, so you can stay inside your IDE. Only added it incase someone want to use it.